recipe/meta.yaml

```diff
@@ -1,45 +1,47 @@
-{% set version = "2.17.7" %}
-{% set min_astroid = "2.15.8" %}
-{% set max_astroid = "2.17.0-dev0" %}
+{% set version = "3.0.0" %}
+{% set min_astroid = "3.0.0" %}
+{% set max_astroid = "3.1.0-dev0" %}
 
 package:
   name: pylint
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pylint/pylint-{{ version }}.tar.gz
-  sha256: f4fcac7ae74cfe36bc8451e931d8438e4a476c20314b1101c458ad0f05191fad
+  sha256: d22816c963816d7810b87afe0bdf5c80009e1078ecbb9c8f2e2a24d4430039b1
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv
   entry_points:
     - pylint = pylint:run_pylint
-    - epylint = pylint:run_epylint
+    - pylint-config = pylint:_run_pylint_config
     - pyreverse = pylint:run_pyreverse
     - symilar = pylint:run_symilar
 
 requirements:
   host:
-    - python >=3.7.2
+    - python >=3.8.0
     - pip
+    - setuptools >=66.1
 
   run:
     - astroid >={{ min_astroid }},<{{ max_astroid }}
     - colorama >=0.4.5
     - isort >=4.2.5,<6
     - mccabe >=0.6,<0.8
     - platformdirs >=2.2.0
-    - python >=3.7.2
+    - python >=3.8.0
     - tomli >=1.1.0
     - tomlkit >=0.10.1
     - typing_extensions >=3.10.0
     # technically...
     # "dill>=0.2;python_version<'3.11'",
     # "dill>=0.3.6;python_version>='3.11'",
+    # "dill>=0.3.7;python_version>='3.12'",
     # but to keep noarch...
-    - dill >=0.3.6
+    - dill >=0.2
```

Comment:
Why did this drop to the lowest lower bound? I thought we would want the highest lower bound to ensure compatibility for all Python versions, so 0.3.7 in this case.

Reply:
yeah, whoops, should be fixed